src/commands/new/glee.ts

```diff
@@ -0,0 +1,42 @@
+import {Flags} from '@oclif/core';
+import { fstat, promises as fPromises } from 'fs';
+import Command from '../../base';
+import { resolve } from 'path';
+import path from 'path';
+import fs from 'fs-extra';
+
+const { writeFile, readFile } = fPromises;
+const fs = require('fs-extra');
+
+export default class NewGlee extends Command {
+  static description = 'Creates a new Glee project';
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+    name: Flags.string({ char: 'n', description: 'name of the project', default: 'glee-project' }),
+  };
+
+  async run() {
+    const { flags } = await this.parse(NewGlee); // NOSONAR
+
+    const projectName = flags.name;
+
+    const PROJECT_DIRECTORY = path.join(process.cwd(), projectName); // cwd-> /Users/pedro.ramos/cli
+    const GLEE_TEMPLATES_DIRECTORY = resolve(__dirname, '../../../create-glee-app/templates/default'); // __dirname-> /Users/pedro.ramos/cli/lib/commands/new
+
+    try {
+      await fPromises.mkdir(PROJECT_DIRECTORY);
+      console.log('Glee project folder created!');
```

Comment:
>  Also, make sure console messages suggest `asyncapi new project` or `asyncapi new glee`, depending on which one has been used.

I guess we could apply your suggestion also to this console message, right? 🤔

Reply:
Apart from that, I think this console message should be removed cause it would be redundant after implementing this other suggestion: https://github.com/asyncapi/cli/pull/418#discussion_r1087651323

Reply:
> Apart from that, I think this console message should be removed since it would be redundant after implementing this other suggestion: [#418 (comment)](https://github.com/asyncapi/cli/pull/418#discussion_r1087651323)

Exactly. Good point 👍